src/franka_plan_runner.cc

```diff
@@ -213,6 +213,12 @@ int FrankaPlanRunner::RunFranka() {
                                   utils::RobotModeToString(current_mode))};
         dexai::log()->error("RunFranka: {}", err_msg);
         comm_interface_->PublishDriverStatus(false, err_msg);
+      } else if (current_mode == franka::RobotMode::kUserStopped) {
+        dexai::log()->error("RunFranka: Robot UStopped!");
+        comm_interface_->PublishBoolToChannel(
+            utils::get_current_utime(),
+            comm_interface_->GetUserStopChannelName(),
+            current_mode == franka::RobotMode::kUserStopped);
```

Comment:
You don't need the ==, you're already in a block it will only hit if user stopped. This can just be `true`